### DISCUSSION TITLE
v0.3.13-beta.0 - Ffmpeg Profile support

### DISCUSSION THREAD

**Discussion by jessedp** [jessedp/tablo-tools-electron#190] [Announcements]
These options can be found under `Settings->Export: Ffmpeg Profiles` . In this first beta, there is only a **single global default setting** that can be set, though you can define as many profiles as you'd like. There are also a number of pre-built profiles. The default (Basic) profile applied still provides the same functionality as existing versions of TabloTools. The intent is to allow them to be applied more with more granularity like Naming Templates can be.

Feedback is appreciated as this is a fairy major feature.

* Add support for Ffmpeg profiles similar to [ffmpeg-commander](https://alfg.dev/ffmpeg-commander/)

<hr /><em>This discussion was created from the release <a href='https://github.com/jessedp/tablo-tools-electron/releases/tag/v0.3.13-beta.0'>v0.3.13-beta.0 - Ffmpeg Profile support</a>.</em>

**Comment by dougK44883** [comment]
> 
> Feedback is appreciated as this is a fairy major feature.
> 
This looks like a great addition!  Again thanks for the continued development with Tablo Tools!   ...I hope users find it beneficial.
  Me, I've read ffmpeg's docs and search and find little _understandable_ information.    It's like "by developers for developers" although I believe I read that about gstreamer, they are similar in some ways.

I found this, not gone, from someone "hacking" their tablo
`ffmpeg -loglevel warning -i "http://tablo0.lan:18080/pvr/$1/pl/playlist.m3u8" -bsf:a aac_adtstoasc -vcodec copy -acodec copy -crf 18 "$2".mp4`
(I added the variables for scripting)  I have little understanding what the switches do, and no documentation really says anything I understand.  It took a lot to fine the loglevel adjustment.

**Comment by JUser69** [comment]
Looks promising!  However how do you apply a template that you have named.  Needs to be before the export seems it should ask if you want to use a different named template (ffmpeg).  Also, on first try I tried adding something to the additional options command line and it seemed to hang up the program.   I do like that your considering so many options (as drop down selections) in the configuration area though.
